src/main/scala/com/trueconnectivity/CommonConfigPlugin.scala

```diff
@@ -126,7 +126,7 @@ object CommonConfigPlugin extends AutoPlugin {
       )
 
     lazy val trueconnectivityCommonSettings: Seq[Def.Setting[_]] = Seq(
-      organization := "com.trueconnectivity",
+      organization in ThisBuild := "com.trueconnectivity",
       scalaVersion := "2.12.10"
```

Comment:
Not sure what the rule is, but for some keys the default scope applies to subproject (as the subproject key has `Zero / * / *` as delegate) and for other keys this is not the case.

F. ex.:
```
sbt:utility-root> inspect core/scalaVersion
[info] Delegates:
[info] 	core / Compile / scalaVersion
[info] 	core / scalaVersion
[info] 	ThisBuild / Compile / scalaVersion
[info] 	ThisBuild / scalaVersion
**[info] 	Zero / Compile / scalaVersion**
[info] 	Global / scalaVersion

sbt:utility-root> inspect core/organization
[info] 	core / organization
[info] 	ThisBuild / organization
[info] 	Global / organization
```